testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -320,6 +322,22 @@ QUnit.module('typing', moduleConfig, () => {
         assert.equal(keyboard.caret().start, 3, 'caret in correct position');
     });
 
+    QUnit.test('autofill input with mask should not throw error (T996580)', function(assert) {
+        const $textEditor = $('#texteditor').dxTextEditor({
+            mask: '+1 (X00) 000-0000',
+            value: '+1'
+        });
+        const $input = $textEditor.find(`.${TEXTEDITOR_INPUT_CLASS}`);
+
+        try {
+            $($input).trigger($.Event('input', { originalEvent: $.Event('input') }));
+
+            assert.ok(true, 'everything is ok');
```

Comment:
A u sure this tests checks that **everything** is ok? :-)

Reply:
just took that message from another test) changed to 'no error logged'